swarmrl/trainers/episodic_trainer.py

```diff
@@ -49,6 +50,9 @@ def perform_rl_training(
                 After how many episodes is the simulation reset.
         load_bar : bool (default=True)
                 If true, show a progress bar.
+        save_episodic_data : bool (default=False)
```

Comment:
That's a really cool thing to have, I would only suggest that it be set default to `True`

Reply:
I set it to `True` and changed the other tests accordingly.